sign_oca/views/sign_oca_request.xml

```diff
@@ -20,7 +20,7 @@
                     <button
                         string="Send"
                         type="object"
-                        states="1_draft"
+                        states="1_draft,0_sent"
```

Comment:
Continuing with https://github.com/OCA/sign/pull/124#discussion_r2329296336, this is NOT what I suggested. The “send” button in the header should remain as it is and should not be changed. A button (an icon that is representative without text) should be added to the signers, and that action should send the email to that recipient, nothing else.

Reply:
I think the “_simple_” thing to do is to separate this part https://github.com/OCA/sign/blob/e1440502718c02f3363c88f679dcc5ade01baaf7/sign_oca/models/sign_oca_request.py#L244 into a specific method in sign.request. signer and call that method to “resend.”